src/components/TodoComponent/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './TodoComponent';
```

Comment:
It's recommended to avoid using wildcard exports. Instead, explicitly import and export the components to maintain consistency and readability 📖.